cuML/src/glm/glm.cu

```diff
@@ -180,29 +180,27 @@ void qnFit(float *X, float *y, int N, int D, int C, bool fit_intercept,
            int linesearch_max_iter, int lbfgs_memory, int verbosity, float *w0,
            float *f, int *num_iters, bool X_col_major, int loss_type) {
 
-  // TODO handles will come from the cuml handle
-  cudaStream_t stream = 0;
-  cublasHandle_t cublas_handle;
-  CUBLAS_CHECK(cublasCreate(&cublas_handle));
+  // TODO cuml will be passed in
+  cumlHandle cuml;
+  cuml.setStream(0);
+
   qnFit(X, y, N, D, C, fit_intercept, l1, l2, max_iter, grad_tol,
         linesearch_max_iter, lbfgs_memory, verbosity, w0, f, num_iters,
-        X_col_major, loss_type, cublas_handle, stream);
-  CUBLAS_CHECK(cublasDestroy(cublas_handle));
+        X_col_major, loss_type, cuml.getImpl());
```

Comment:
Minor knitpick: Similar to @jirikraus's comments [here](https://github.com/rapidsai/cuml/pull/482#pullrequestreview-227762358), please use the following interface to any functions inside cuML, while exposing `cumlHandle`:
1. `cumlHandle_impl` is the first argument to be passed
2. `cudaStream_t` should be passed explicitly as the last argument
If you need an example, dbscan clean-up by Vinay [here](https://github.com/rapidsai/cuml/blob/branch-0.7/cuML/src/dbscan/dbscan.h) should be helpful.